README.md

```diff
@@ -81,18 +81,57 @@ tsx        [✗] not installed
 
 And now you should be able to use every functionality `nvim-treesitter` provides!
 
+## Commands
+
+Each feature can be enabled or disabled by different means:
+```vim
+:TSBufEnable {module} " enable module on current buffer
+:TSBufDisable {module} " disable module on current buffer
+:TSEnableAll {module} {ft}? " enable module on every buffer. If filetype is specified, enable only for this filetype.
+:TSDisableAll {module} {ft}? " disable module on every buffer. If filetype is specified, disable only for this filetype.
```

Comment:
Maybe changing `{ft}?` to `[{ft}]` would be more clear as it is the standard way to refer to optionnal arguments in vim.

Reply:
ok i though it was for list arguments